packages/trader/src/Modules/Trading/Components/Form/Purchase/contract-info.jsx

```diff
@@ -79,10 +79,8 @@ const ContractInfo = ({
         if (is_turbos) {
             return (
                 <Localize
-                    i18n_default_text='<0>{{title}}</0> {{message}}'
-                    components={[<Text key={0} weight='bold' size='xxs' />]}
+                    i18n_default_text='{{message}}'
```

Comment:
This gets translated in BE right? We don't need localize here.

Reply:
I have no way to check what language it comes in at the moment. They disabled Turbos from backoffice due to backend bug

Reply:
It doesn't matter actually, this localize won't work. Because variables are used inside to pass dynamic values into a string and it doesn't translate the content of the variables.

Reply:
I suggest to remove it if it's confirmed that it is received from BE, and BE can handle translation if it's not there. Btw, you may still test it on any QA box if it's deployed and enabled.